Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/FilteredLogFileTest.cs

```diff
@@ -41,7 +41,7 @@ public void TestFilter1()
 				using (FilteredLogFile filtered = file.AsFiltered(_scheduler, null, Filter.Create("info")))
 				{
 					filtered.Property(x => x.Count).ShouldEventually().Be(5, TimeSpan.FromSeconds(5));
-					filtered.GetValue(LogFileProperties.StartTimestamp).Should().Be(new DateTime(2015, 10, 7, 19, 50, 58, 982));
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.